packages/api/src/external/ehr/elation/command/process-patients-from-appointments.ts

```diff
@@ -0,0 +1,255 @@
+import ElationApi, { ElationEnv } from "@metriport/core/external/elation/index";
+import { executeAsynchronously } from "@metriport/core/util/concurrency";
+import { out } from "@metriport/core/util/log";
+import { capture } from "@metriport/core/util/notifications";
+import { errorToString, MetriportError } from "@metriport/shared";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { getCxMappingsBySource } from "../../../../command/mapping/cx";
+import { EhrSources, getLookforwardTimeRange } from "../../shared";
+import { getElationClientKeyAndSecret, getElationEnv, MAP_KEY_SEPARATOR } from "../shared";
+import { getPatientIdOrFail } from "./get-patient";
+
+dayjs.extend(duration);
+
+const delayBetweenPracticeBatches = dayjs.duration(0, "seconds");
+const lookforward = dayjs.duration(14, "days");
+const parallelPractices = 10;
+const parallelPatients = 2;
+
+type PatientAppointment = {
+  cxId: string;
+  elationPracticeId: string;
+  elationPatientId: string;
+};
+
+export async function processPatientsFromAppointments(): Promise<void> {
```

Comment:
This function is called on the endpoint asynchronously and it doesn't have a `.catch()` there.

This is fine, as long as we have a try/catch around the whole code of the function - otherwise  we won't have any error handing in case we get an unexpected error.

Reply:
Done.